.vscode/tasks.json

```diff
@@ -8,18 +8,23 @@
 
 // A task runner that calls a custom npm script that compiles the extension.
 {
-  "version": "0.1.0",
-  "command": "npm",
-  "isShellCommand": true,
-  "suppressTaskName": true,
+  "version": "2.0.0",
   "tasks": [
     {
-      "taskName": "build",
+      "label": "build",
+      "type": "shell",
+      "command": "npm",
       "args": ["run", "compile"],
-      "echoCommand": true,
-      "isBuildCommand": true,
-      "showOutput": "always",
-      "problemMatcher": "$tsc-watch"
+      "group": "build",
+      "problemMatcher": "$tsc-watch",
+      "presentation": {
+        "echo": true,
+        "reveal": "always",
+        "focus": false,
+        "panel": "shared",
+        "showReuseMessage": true,
+        "clear": false

```

Comment:
I don't see these properties listed on https://code.visualstudio.com/docs/editor/tasks-appendix is there documentation for them somewhere else?

Reply:
These additions to the `presentation` property just customize the appearance/behavior of the terminal when running tasks.

- `showReuseMessage` controls whether the terminal shows the message "Terminal will be reused by tasks, press any key to close it."
- `clear` controls whether the terminal is cleared before that particular task is run.

Both of these are optional with the current configuration, because they are set to default behavior.

The full documentation on `presentation` is in [this chapter](https://code.visualstudio.com/docs/editor/tasks) of the VS Code User Guide, in the section titled "Output Behavior."